config/clusters/iam.tf

```diff
@@ -218,6 +218,44 @@ data "aws_iam_policy_document" "rules_s3_access" {
   }
 }
 
+# Test-infra repository
+
+module "test-infra_s3_role" {
+  source    = "terraform-aws-modules/iam/aws//modules/iam-github-oidc-role"
+  version = "5.10.0"
+  create = true
+  subjects = [
+    "falcosecurity/test-infra:ref:refs/heads/master"
+  ]
+  policies = {
+    test-infra_s3_access = "${aws_iam_policy.test-infra_s3_access.arn}"
+  }
+}
+
+resource "aws_iam_policy" "test-infra_s3_access" {
+  name_prefix = "github_actions-test-infra-s3"
+  description = "GitHub actions S3 access policy for test-infra update-drivers-website workflow"
+  policy      = data.aws_iam_policy_document.test-infra_s3_access.json
+}
+
+data "aws_iam_policy_document" "test-infra_s3_access" {
+  statement {
+    sid    = "UploadTestInfraS3Access"
+    effect = "Allow"
+    actions = [
+      "s3:PutObject",
+      "s3:GetObjectAcl",
+      "s3:GetObject",
+      "s3:DeleteObject",
+      "s3:PutObjectAcl"
+    ]
+    resources = [
+      "arn:aws:s3:::falco-distribution/driver/site/*",
```

Comment:
We cannot escape `falco-distribution/driver/site` ;) Thus we should not be able to make any damage (eg: deleting drivers)!